capstone/yarn.lock

```diff
@@ -9245,6 +9245,11 @@ vue-router@^3.0.2:
   resolved "https://registry.yarnpkg.com/vue-router/-/vue-router-3.4.9.tgz#c016f42030ae2932f14e4748b39a1d9a0e250e66"
   integrity sha512-CGAKWN44RqXW06oC+u4mPgHLQQi2t6vLD/JbGRDAXm0YpMv0bgpKuU5bBd7AvMgfTz9kXVRIWKHqRwGEb8xFkA==
 
+vue-select@^3.11.2:
```

Comment:
@bensteinberg vue-select is installed, oddly

Reply:
I wonder if this is an artifact of an old Docker image cache -- if so, a subsequently-merged PR that busts the cache will pass. We don't have one yet.

Reply:
@bensteinberg #1771 might. I installed something with pip and bumped the version, if that helps.